### DISCUSSION TITLE
Importing GPX with Waypoints, navigating as intended

### DISCUSSION THREAD

**Discussion by outdoor720** [osmandapp/OsmAnd#19969] [Q&A]
I'm importing a gpx file for the ottawa ride for dad 2024.  It does not appear to import the poker stops(waypoints)  as well does not seem to follow the intended route exactly.  

How am i supposed to adjust for this?  Do i need to add intermediate stops?  Do i have to attach roads?  the min is 5M, not even sure what this does.

If i say route from my position to route start, it does not appear to show the remainder of the route.  Will it once i arrive at start of route?

**Comment by DmitryAlexei** [comment]
@outdoor720 
Was the problem resolved? 
Some useful information about using .gpx files for navigation you can find in docs
https://osmand.net/docs/user/navigation/setup/gpx-navigation
